internal/dataplane/parser/translate_httproute.go

```diff
@@ -270,8 +270,7 @@ func generateKongRoutesFromHTTPRouteMatches(
 
 	// if the redirect filter has not been set, we still need to set the route plugins
 	if !hasRedirectFilter {
-		plugins := translators.GeneratePluginsFromHTTPRouteFilters(filters, "", tags)
-		r.Plugins = append(r.Plugins, plugins...)
+		translators.ConvertFiltersToPlugins(&r, filters, "", tags)
```

Comment:
This returns an `error`. Why not check it here? Same elsewhere

Reply:
my bad, the error is now checked everywhere